.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.0
+jobs:
+  go-vet-and-test:
+    docker:
+      - image: circleci/golang:1.13.3
+    working_directory: /go/src/github.com/moderepo/device-sdk-go
```

Comment:
I'm not so sure, but we may not have to specify working_directory because we now are using go module.

Reply:
Cool. Removed it and it ran locally fine.